### PR TITLE
Initial commit for 4.22

### PR DESCRIPTION
**Pull request by andygup**
Updates CHANGELOG and README.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,114 +10,121 @@ The following classes, methods, properties and events have been deprecated for a
...
 | Class/Property/Method/Event | Alternate option | Version deprecated |
 |----------|-------------|--------------------|
+| `BasemapLayerList.statusIndicatorsVisible` | `BasemapLayerList.visibleElements.statusIndicators` | 4.15 |
 | `Bookmarks.bookmarkCreationOptions` | `Bookmarks.defaultCreateOptions` | 4.18 |
+| `esri/tasks/ImageServiceIdentifyTask` | `imageService.identify` | 4.18 |
+| `esri/tasks/support/ImageServiceIdentifyResult` | `ImageIdentifyResult` | 4.18 |
+| `esri/tasks/support/ImageServiceIdentifyParameters` | `ImageIdentifyParameters` | 4.18 |
+| `FeatureForm.description` and `FeatureFormViewModel.description`  | `FormTemplate.description` set either on the `FeatureForm` or `FeatureLayer.formTemplate` | 4.18 |
+| `FeatureForm.fieldConfig` and `FeatureFormViewModel.fieldConfig` | `FieldElement/GroupElement` set in the `FormTemplate.elements` | 4.16 |
+| `FeatureForm.title` and `FeatureFormViewModel.title` | `FormTemplate.title` set either on the `FeatureForm` or `FeatureLayer.formTemplate`  | 4.18 |
```
Comment (annelfitz) [inline CHANGELOG.md:20]: I would remove these. they're breaking changes in 4.21 not 4.22

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,114 +10,121 @@ The following classes, methods, properties and events have been deprecated for a
...
+*    Task deprecated since version 4.20.
+*    TimeSlider.values deprecated since version 4.20. Use timeExtent instead.
+*    TimeSliderViewModel.values deprecated since version 4.20. Use timeExtent instead.
+*    TrimExtendParameters deprecated since version 4.20. Use TrimExtendParameters instead.
+*    widget.renderable deprecated since version 4.19. All properties are automatically tracked now and don't need to be decorated with this decorator.
+*    `SmartMapping.params.basemap` deprecated since version 4.13. Use `view` instead.
+*    The light-blue, dark-blue, light-green, dark-green, light-purple, dark-purple, light-red, dark-red are deprecated since 4.19. Please use `light` or `dark` instead, or create your own theme.
+
 
 BELOW ARE THE MANUALLY ADDED DEPRECATED CLASSES, PROPERTIES, METHODS, EVENTS
```
Comment (annelfitz) [inline CHANGELOG.md:126]: this should not be here

File: CHANGELOG.md [hunk 3]
```diff
@@ -10,114 +10,121 @@ The following classes, methods, properties and events have been deprecated for a
...
+| `FeatureForm.title` and `FeatureFormViewModel.title` | `FormTemplate.title` set either on the `FeatureForm` or `FeatureLayer.formTemplate`  | 4.18 |
 
 Please refer to the [Breaking changes](https://developers.arcgis.com/javascript/latest/breaking-changes/) guide topic for a complete list of breaking changes across all releases of the 4x API.
 
 ## Bug fixes and enhancements
 
-- BUG-000141284: Fixed an issue with [WMSLayer](https://developers.arcgis.com/javascript/latest/api-reference/esri-layers-WMSLayer.html) where custom parameters would be added to the GetLegendGraphic request with a `?` instead of an `&`, causing the request to fail.
-- Fixed an issue where layer effects were not being applied to ImageryLayer.
-- Fixed an issue where VectorFieldRenderer would not update when used with the TimeSlider widget.
+* TBD
```
Comment (annelfitz) [inline CHANGELOG.md:26]: I added two bug fixes to the release notes this morning. not sure when `/next` was published but they might be able to be included here

File: CHANGELOG.md [hunk 4]
```diff
@@ -10,114 +10,121 @@ The following classes, methods, properties and events have been deprecated for a
...
 
 ## Deprecations
 The following are deprecated and will be removed in a future release. For anything deprecated in 4.20 and earlier, additional information and links are in the [release notes](https://developers.arcgis.com/javascript/latest/release-notes/#deprecated-classes-properties-methods-events).
```
Comment (annelfitz) [inline CHANGELOG.md:29]:
```suggestion
The following are deprecated and will be removed in a future release. For anything deprecated in 4.21 and earlier, additional information and links are in the [release notes](https://developers.arcgis.com/javascript/latest/release-notes/#deprecated-classes-properties-methods-events).
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -10,114 +10,121 @@ The following classes, methods, properties and events have been deprecated for a
...
+*    The light-blue, dark-blue, light-green, dark-green, light-purple, dark-purple, light-red, dark-red are deprecated since 4.19. Please use `light` or `dark` instead, or create your own theme.
+
 
 BELOW ARE THE MANUALLY ADDED DEPRECATED CLASSES, PROPERTIES, METHODS, EVENTS
 
-*	decorators.cast(classFunction) deprecated since version 4.14. Parameter decorators won't be supported by JavaScript decorators.
-*	LabelClass.labelExpressionInfo.value deprecated since version 4.5. Use expression instead.
-*	SceneView.constraints.collision deprecated since version 4.8. Use Ground.navigationConstraint instead.
-*	SmartMapping.params.basemap deprecated since version 4.13. Use view instead.
-*	The light-blue, dark-blue, light-green, dark-green, light-purple, dark-purple, light-red, dark-red are deprecated since 4.19. Please use light or dark instead, or create your own theme.
```
Comment (annelfitz) [inline CHANGELOG.md:121]: these should still be included I think

File: CHANGELOG.md (hunk 2 above)
Comment (annelfitz) [reply CHANGELOG.md:126]: it should be one cohesive list, not separated by manual vs automatic

**Review by annelfitz — CHANGES_REQUESTED** [review]
some minor changes needed. otherwise, looks good

File: CHANGELOG.md (hunk 1 above)
Comment (andygup) [reply CHANGELOG.md:20]: SGTM.

File: CHANGELOG.md (hunk 2 above)
Comment (andygup) [reply CHANGELOG.md:126]: Hmmm, okay. I was just copying what we had left over from the 4.21.

File: CHANGELOG.md (hunk 3 above)
Comment (andygup) [reply CHANGELOG.md:26]: np, I can copy them over.

File: CHANGELOG.md (hunk 5 above)
Comment (andygup) [reply CHANGELOG.md:121]: They seem to all be there. I copied over the 4.21 release notes, so everything got included and then alphabetized.

File: CHANGELOG.md (hunk 4 above)
Comment (andygup) [reply CHANGELOG.md:29]: Done, included in my local PR.

File: CHANGELOG.md (hunk 3 above)
Comment (andygup) [reply CHANGELOG.md]: Done.

File: CHANGELOG.md (hunk 2 above)
Comment (andygup) [reply CHANGELOG.md]: Done.

**Review by annelfitz — APPROVED** [review]

File: CHANGELOG.md (hunk 5 above)
Comment (andygup) [reply CHANGELOG.md:121]: Resolved and did some minor reorg.

**Pull request merged**